### ISSUE TITLE
♿ AVT3 ♿ Dropdown - multiple issues

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1589]
User tested with latest version of JAWS 2018, Windows 7, Chrome latest version

Three issues:
1) All of the dropdown's labels are pronounced when the user first selects the component
2) The user has no indication that they've selected a particular dropdown item
3) There is no obvious declaration of the components intent or use.
_"It is not obvious that it is, or should function as a dropdown"_

**Comment by elizabethsjudd** [comment]
When running this through DAP the list is technically not visually hidden via (`visibility: hidden` or `display: none`) so we are getting multiple color contrast violations.

FYI, WH is working on this component now and we'll provide a `migrate-to-11.x.md` file for the changes we made

**Comment by dakahn** [comment]
Closing this. We can track accessibility work on the dropdown at #1147

**Comment by asudoh** [comment]
@elizabethsjudd Could you share what it looks like as of now so we can see our change fits well in our codebase? Having a draft PR will be great - Thanks a lot for working on this!

**Comment by elizabethsjudd** [comment]
Here is may latest PR in to WH https://github.ibm.com/Watson-Health/design-library/pull/3024 (keep in mind this is still a WIP component and more changes are expected).  We have been documenting our deviations from Carbon to help clarify the changes we make.  You can see them in this file: https://github.ibm.com/Watson-Health/design-library/blob/a6dde364e82b9c6e779c10e496ca1bb335d63298/packages/pattern-library/src/components/dropdown/migrate-to-11.x.md

Note: We recognize these would be breaking changes and are hoping that these changes would go in to Carbon 11

**Comment by asudoh** [comment]
Thanks @elizabethsjudd for sharing, especially for the list of changes you have made. It'll helps us port your changes over to `carbon-components`, ensuring that the porting effort makes your changes aligned to the existing structure of `carbon-components` codebase. Thanks again!
